python/mxnet/_numpy_op_doc.py

```diff
@@ -58,31 +55,53 @@ def _np_reshape(a, newshape, order='C'):
 
 
 def _np_ones_like(a):
-    """Return an array of ones with the same shape and type as a given array.
+    """
+    ones_like(a)
 
+    Return an array of ones with the same shape and type as a given array.
+    
     Parameters
     ----------
-    a : ndarray
+    a : array_like
```

Comment:
``ndarray`` might be better? Does np_ones_like and np_zeros_like support python lists / tuples?

Reply:
Changed. Thx.

Reply:
@ckt624 Please remember to push your updates.